### PR TITLE
Action bar icon visibility

### PR DESCRIPTION
**Pull request by Jason-Lopez8**
Able to select/unselect the following : 

- Text Appearance Icon
- Search Icon
- Book Collection selector Icons
- Audio Icon

### CODE REVIEW

File: src/lib/components/Navbar.svelte [hunk 1]
```diff
@@ -89,86 +89,97 @@ The navbar component.
...
+            </Dropdown>
+        {/if}
     </div>
     <div class="dy-navbar-end fill-base-content">
         <!-- Mute/Volume Button -->
-        <button class="dy-btn dy-btn-ghost dy-btn-circle">
-            <label class="dy-swap">
-                <!-- this hidden checkbox controls the state -->
-                <input type="checkbox" bind:checked={$audioActive} />
+        {#if $globalConfig.mainFeatures["audio-turn-on-at-startup"]}
```
Comment (chrisvire) [inline src/lib/components/Navbar.svelte:154]: The visibility of the audio button is the nav bar is based on whether there is audio in the current chapter.  The "audio-turn-on-at-startup" controls whether the audio toolbar at the bottom displays when the app is launched (if the current chapter has audio).  If this is unchecked, then the audio toolbar is not displayed if the current chapter has audio.

File: src/lib/components/Navbar.svelte [hunk 2]
```diff
@@ -89,86 +89,97 @@ The navbar component.
...
+            </a>
+        {/if}
         <!-- Text Appearance Options Menu -->
-        <Dropdown>
-            <svelte:fragment slot="label">
-                <TextAppearanceIcon />
-            </svelte:fragment>
-            <!-- TODO: implement text appearance options -->
-        </Dropdown>
+        {#if $globalConfig.mainFeatures["text-font-size-slider"] && $globalConfig.mainFeatures[ "text-line-height-slider"] }
```
Comment (chrisvire) [inline src/lib/components/Navbar.svelte:176]: This should be an || (or) not an && (and).  Also, need to check whether there is more than one theme enabled.

File: src/routes/search/+page.svelte [hunk 3]
```diff
@@ -4,9 +4,14 @@
     import { refs } from '$lib/data/stores';
     import { query } from '$lib/scripts/query';
     import { postQueries, queries } from 'proskomma-tools';
+
```
Comment (chrisvire) [inline src/routes/search/+page.svelte:7]: Why is this file changed?  It seems to just have extra newlines added without any real changes.

**Review by chrisvire — CHANGES_REQUESTED** [review]

File: src/lib/components/Navbar.svelte [hunk 4]
```diff
@@ -89,86 +89,97 @@ The navbar component.
...
-                            component: SelectGrid,
-                            props: { options: Object.keys(chapters[nextRef.chapter]) }
-                        }
-                    }}
-                    active="Chapter"
-                    on:menuaction={navigateReference}
-                />
-            </svelte:fragment>
-        </Dropdown>
+        {#if $globalConfig.mainFeatures["show-chapter-numbers"]}
```
[the remark this answers is not in the document]
Comment (Jason-Lopez8) [reply src/lib/components/Navbar.svelte]: Changed this

File: src/lib/components/Navbar.svelte [hunk 5]
```diff
@@ -17,9 +17,9 @@ The navbar component.
         VerseByVerseIcon
     } from '$lib/icons';
     import { catalog } from '$lib/data/catalog';
-    import { globalConfig } from '$lib/data/stores';
     import { audioActive, refs } from '$lib/data/stores';
     import { onDestroy } from 'svelte';
+    import { globalConfig } from '$lib/data/stores';
```
[the remark this answers is not in the document]
Comment (Jason-Lopez8) [reply src/lib/components/Navbar.svelte]: Changed this

File: src/routes/search/+page.svelte (hunk 3 above)
Comment (Jason-Lopez8) [reply src/routes/search/+page.svelte:7]: I can't explain why, could it be when I first used prettier it could've added those lines?

File: src/lib/components/Navbar.svelte (hunk 2 above)
Comment (Jason-Lopez8) [reply src/lib/components/Navbar.svelte]: Referring to the theme would it be another comparison to text-font-size-slider and text-line-height-slider?

File: src/lib/components/Navbar.svelte [hunk 6]
```diff
@@ -89,86 +88,98 @@ The navbar component.
...
-                            props: { options: Object.keys(chapters[nextRef.chapter]) }
-                        }
-                    }}
-                    active="Book"
-                    on:menuaction={navigateReference}
-                />
-            </svelte:fragment>
-        </Dropdown>
+        {#if $globalConfig.mainFeatures['book-select'] === 'grid' || $globalConfig.mainFeatures['book-select'] === 'list'}
+            <Dropdown>
```
Comment (chrisvire) [inline src/lib/components/Navbar.svelte:92]: Please change this so that it is:

```
{#if  $globalConfig.mainFeatures['book-select'] === 'grid' }
<Dropdown>
...
</Dropdown>
{:else if $globalConfig.mainFeatures['book-select'] === 'list' }
<! -- TODO: Add List selector -->
{/if}

File: src/lib/components/Navbar.svelte [hunk 7]
```diff
@@ -89,86 +88,98 @@ The navbar component.
...
-                            component: SelectGrid,
-                            props: { options: Object.keys(chapters[nextRef.chapter]) }
-                        }
-                    }}
-                    active="Chapter"
-                    on:menuaction={navigateReference}
-                />
-            </svelte:fragment>
-        </Dropdown>
+        {#if $globalConfig.mainFeatures['show-chapter-number-on-app-bar']}
```
Comment (chrisvire) [inline src/lib/components/Navbar.svelte:127]: Is there a separate issue to implement 'show-chapter-selector' and 'show-chapter-selector-after-book'?

File: src/lib/components/Navbar.svelte (hunk 2 above)
Comment (chrisvire) [reply src/lib/components/Navbar.svelte]: @Jason-Lopez8 You need to get a count of $globalConfig.themes that are enabled.

**Review by chrisvire — CHANGES_REQUESTED** [review]
Minor change with book selector
Need theme count on text appearance

File: src/routes/search/+page.svelte (hunk 3 above)
Comment (chrisvire) [reply src/routes/search/+page.svelte:7]: Prettier doesn't normally add blank lines.  Please remove this file from the PR.

File: src/lib/components/Navbar.svelte (hunk 6 above)
Comment (Jason-Lopez8) [reply src/lib/components/Navbar.svelte:101]: done

File: src/lib/components/Navbar.svelte (hunk 7 above)
Comment (Jason-Lopez8) [reply src/lib/components/Navbar.svelte:138]: Yes, checked in with Jake he said there was

File: src/lib/components/Navbar.svelte [hunk 8]
```diff
@@ -173,7 +184,7 @@ The navbar component.
             </a>
         {/if}
         <!-- Text Appearance Options Menu -->
-        {#if $globalConfig.mainFeatures['text-font-size-slider'] || $globalConfig.mainFeatures['text-line-height-slider']}
+        {#if $globalConfig.mainFeatures['text-font-size-slider'] || $globalConfig.mainFeatures['text-line-height-slider'] || countThemesNormal || countThemesDark || countThemesSepia}
             <Dropdown>
                 <svelte:fragment slot="label">
                     <TextAppearanceIcon />
```
Comment (Jason-Lopez8) [inline src/lib/components/Navbar.svelte:190]: I added the above on lines 53-61 because it made more sense to me to have these variables and do the comparison that way, if you want me to changed it to the count way you said I can.

File: src/routes/search/+page.svelte (hunk 3 above)
Comment (Jason-Lopez8) [reply src/routes/search/+page.svelte]: Deleted file

Comment (Jason-Lopez8) [reply src/routes/search/+page.svelte:7]: I undid this realizing I did it incorrectly. reverted file back

**Comment by chrisvire** [comment]
@Jason-Lopez8 There were significant changes to the structure.  This will need to be redone.  Sorry.  Please contact me via Discord and I can help you with the transition.

**Comment by chrisvire** [comment]
Will be done in #80
